lib/mix/nerves/utils.ex

```diff
@@ -194,18 +194,46 @@ defmodule Mix.Nerves.Utils do
     end
   end
 
-  def get_wsl_paths(file) do
-    {win_path, 0} = System.cmd("cmd.exe", ["/c", "cd"])
-    win_path = String.trim(win_path) <> "\\#{file}"
+  def has_wslpath?() do
+    System.find_executable("wslpath") != nil
+  end
 
-    drive_letter =
-      Regex.run(~r/(.*?):\\/, win_path)
-      |> Enum.at(1)
-      |> String.downcase()
+  def is_wsl_path_accessible?(file) do
```

Comment:
One more function to remove the `is_` prefix. Fwiw, here's the logic: https://github.com/elixir-lang/elixir/blob/master/lib/elixir/pages/Naming%20Conventions.md#is_-prefix-is_foo